cmd/maya-apiserver/cstor-operator/cspc/start.go

```diff
@@ -67,6 +69,13 @@ func Start() error {
 
 	kubeInformerFactory := kubeinformers.NewSharedInformerFactory(kubeClient, time.Second*30)
 	spcInformerFactory := informers.NewSharedInformerFactory(openebsClient, time.Second*30)
+	// Build() fn of all controllers calls AddToScheme to adds all types of this
+	// clientset into the given scheme.
+	// If multiple controllers happen to call this AddToScheme same time,
+	// it causes panic with error saying concurrent map access.
+	// This lock is used to serialize the AddToScheme call of all controllers.
+	controllerMtx.Lock()
```

Comment:
This code is being repeated everywhere. Please have this at one place only.
Locking should be done only at the place that throws issue. In this case it has to be at `AddToScheme`